test/unit/select_plan_test.lua

```diff
@@ -44,6 +44,25 @@ g.before_all = function()
         unique = false,
         if_not_exists = true,
     })
+
+    -- is used for multipart primary index tests
+    local coord_space = box.schema.space.create('coord', {
+        format = {
+              {name = 'x', type = 'unsigned'},
+              {name = 'y', type = 'unsigned'},
+              {name = 'bucket_id', type = 'unsigned'},
+        },
+        if_not_exists = true,
+    })
+    --primary index
```

Comment:
Here you add a space between "--" and "sentence" (https://github.com/tarantool/crud/pull/90/files#diff-3861a09acb51bc4ef85b8766f9a3a12b055062cb9c8a7fdedface313354d5ccaR27) why don't you do it for this line?